openfe/tests/setup/test_network_planning.py

```diff
@@ -244,6 +244,83 @@ def scorer(mapping):
             scorer=scorer
         )
 
+@pytest.fixture(scope='session')
+def generate_minimal_redundant_network(toluene_vs_others):
+    toluene, others = toluene_vs_others
+    mappers = [BadMapper(), openfe.setup.atom_mapping.LomapAtomMapper()]
+
+    def scorer(mapping):
+        return len(mapping.componentA_to_componentB)
+
+    network = openfe.setup.ligand_network_planning.generate_generate_minimal_redundant_network(
+        ligands=others + [toluene],
+        mappers=mappers,
+        scorer=scorer
+    )
+    return network
+
+def test_generate_minimal_redundant_network(generate_minimal_redundant_network, toluene_vs_others):
+    tol, others = toluene_vs_others
+    assert len(generate_minimal_redundant_network.nodes) == len(others) + 1
```

Comment:
is it not possible to also test the number of edges? Since this is minimal and "next-minimal" spanning trees, it should be `2 * (n_nodes - 1)`, right?